Index: Grive-Debian-packaging/libgrive/CMakeLists.txt
===================================================================
--- Grive-Debian-packaging.orig/libgrive/CMakeLists.txt	2012-06-06 02:06:08.937513308 +0200
+++ Grive-Debian-packaging/libgrive/CMakeLists.txt	2012-06-06 02:06:49.242151330 +0200
@@ -68,6 +68,7 @@
 )
 
 add_library( grive SHARED ${LIBGRIVE_SRC} ${OPT_SRC} )
+add_library( grive-static STATIC ${LIBGRIVE_SRC} ${OPT_SRC} )
 
 target_link_libraries( grive
 	${CURL_LIBRARIES}
@@ -84,12 +85,15 @@
     SOVERSION 0 VERSION ${GRIVE_VERSION}
 )
 
+set_target_properties(grive grive-static PROPERTIES OUTPUT_NAME grive)
+
 if ( LIB_INSTALL_DIR )
 else()
 	set ( LIB_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/lib )
 endif()
 
 install(TARGETS grive LIBRARY DESTINATION ${LIB_INSTALL_DIR} )
+install(TARGETS grive-static ARCHIVE DESTINATION ${LIB_INSTALL_DIR} )
 install(FILES ${DRIVE_HEADERS} DESTINATION include/grive/drive)
 install(FILES ${PROTOCOL_HEADERS} DESTINATION include/grive/protocol)
 install(FILES ${UTIL_HEADERS} DESTINATION include/grive/util)
